### DISCUSSION TITLE
Merge Confidence badges not working - 504 gateway timeout

### DISCUSSION THREAD

**Discussion by CynanX** [renovatebot/renovate#29100] [Request Help]
### What would you like help with?

I think I found a bug

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

GitLab, Renovate v37-full

### Please tell us more about your question or problem

The merge confidence based are not displaying.
When I edit the MR and can a URL out and try to go to it directly I see a CloudFront error, 504 Gateway Timeout.

E.g. URL: [renovatea](https://badges.renovateapi.com/packages/npm/angular-datatables/17.1.0/adoption-slim)

This occurs on my work laptop which has corporate firewall, and on my own laptop which has direct internet access.

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by rarkins** [comment]
That's an old server/URL for badges, which preset or package rule are you using?

**Comment by CynanX** [comment]
Ah thank you! That's pointed me in the right direction and I can see in our config. we are pointing to https://github.com/whitesource/merge-confidence which is deprecated, and the newer docs say to use `mergeConfidence:all-badges`.
Thanks again.

**Reply by rarkins** [reply]
FYI we've updated those definitions to point to the new server too: https://github.com/whitesource/merge-confidence/blob/main/beta.json
